### PR TITLE
Add qualification and contract tests

### PR DESCRIPTION
**Pull request by Leits**
Для запуску потребує `arctifact.yaml` у op_robot_tests/tests_files/datа.

`tender test -> (auction) ->  qualification test -> contract test`

[Review on Reviewable](https://reviewable.io/reviews/openprocurement/robot_tests/73)

### CODE REVIEW

File: op_robot_tests/tests_files/brokers/openprocurement_client.robot [hunk 1]
```diff
@@ -336,6 +336,59 @@ Library  openprocurement_client_helper.py
...
+  ...      Find tender using uaid, get data from confirm_supplier and call patch_award
+  ...      [Return] Nothing
+  [Arguments]  ${username}  ${tender_uaid}  ${award_num}
+  ${tender}=  Пошук тендера по ідентифікатору  ${username}  ${tender_uaid}
+  ${award}=  create_data_dict  data.status  active
+  Set To Dictionary  ${award.data}  id  ${tender.data.awards[${award_num}].id}
+  ${reply}=  Call Method  ${USERS.users['${username}'].client}  patch_award  ${tender}  ${award}
+  Log  ${reply}
+
+Дискваліфікація постачальника
```
Comment (selurvedu) [inline op_robot_tests/tests_files/brokers/openprocurement_client.robot]: Чому не `Дискваліфікувати постачальника`?

File: op_robot_tests/tests_files/keywords.robot [hunk 2]
```diff
@@ -83,6 +88,14 @@ Get Broker Property By Username
   ${broker_name}=  Get Variable Value  ${USERS.users['${username}'].broker}
   Run Keyword And Return  Get Broker Property  ${broker_name}  ${property}
 
+Завантажити дані про тендер
+  ${file_path}=  Get Variable Value  ${ARTIFACT_FILE}  artifact.yaml
+  ${ARTIFACT}=  load_initial_data_from  ${file_path}
+  Set To Dictionary  ${USERS.users['${tender_owner}']}  access_token  ${ARTIFACT.access_token}
```
Comment (selurvedu) [inline op_robot_tests/tests_files/keywords.robot]: Це зламається для брокерів, які не встановлюють access_token. Вище є перевірка, а тут нема.

File: op_robot_tests/tests_files/brokers/openprocurement_client.robot (hunk 1 above)
Comment (selurvedu) [inline op_robot_tests/tests_files/brokers/openprocurement_client.robot]: BTW, де воно буде використовуватись?

File: op_robot_tests/tests_files/contract.robot [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Suite Setup     Test Suite Setup
+Suite Teardown  Test Suite Teardown
+
+*** Variables ***
+${role}         viewer
+${broker}       Quinta
+
+
+*** Test Cases ***
+Пошук позапорогового однопредметного тендера по ідентифікатору
```
Comment (selurvedu) [inline op_robot_tests/tests_files/contract.robot]: `позапорогового однопредметного тендера` → `закупівлі`

File: op_robot_tests/tests_files/contract.robot [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  ${usernames}=  Create List  ${viewer}  ${tender_owner}
+  :FOR  ${username}  IN  @{usernames}
+  \   ${resp}=  Викликати для учасника  ${username}  Пошук тендера по ідентифікатору   ${TENDER['TENDER_UAID']}
+  Log  ${resp}
+
+##############################################################################################
+#             CONTRACT
+##############################################################################################
+
+Можливість укласти угоду для прямої закупівлі
```
Comment (selurvedu) [inline op_robot_tests/tests_files/contract.robot]: `s/прямої //`

File: op_robot_tests/tests_files/contract.robot [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  [Tags]  ${USERS.users['${tender_owner}'].broker}: Можливість укласти угоду для прямої закупівлі
+  ...  tender_owner
+  ...  ${USERS.users['${tender_owner}'].broker}
+  Викликати для учасника  ${tender_owner}  Підтвердити підписання контракту  ${TENDER['TENDER_UAID']}  1
+
+
+Відображення статусу підписаної угоди з постачальником прямої закупівлі
```
Comment (selurvedu) [inline op_robot_tests/tests_files/contract.robot]: `s/прямої //`

File: op_robot_tests/tests_files/qualification.robot [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+Suite Setup     Test Suite Setup
+Suite Teardown  Test Suite Teardown
+
+*** Variables ***
+${role}         viewer
+${broker}       Quinta
+
+
+*** Test Cases ***
+Пошук позапорогового однопредметного тендера по ідентифікатору
```
Comment (selurvedu) [inline op_robot_tests/tests_files/qualification.robot]: `s/позапорогового однопредметного тендера/закупівлі/`

**Comment by selurvedu** [comment]
Reviewed 3 of 3 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/openprocurement/robot_tests/73)*

File: op_robot_tests/tests_files/keywords.robot (hunk 2 above)
Comment (selurvedu) [inline op_robot_tests/tests_files/keywords.robot]: А, бачу.

Resolved in #79.

**Comment by Leits** [comment]
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

*[op_robot_tests/tests_files/contract.robot, line 13 \[r2\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/73#-KCRiVu_aB2LDIPt4-7b-r2-13:-KCUqW_Qnz5lfScXI3bV:118781018) ([raw file](https://github.com/openprocurement/robot_tests/blob/ebc5124ba6a742d4ed6d2c58467e1bd5064fa569/op_robot_tests/tests_files/contract.robot#L13)):*
Можливість знайти закупівлю по ідентифікатору

---

*[op_robot_tests/tests_files/contract.robot, line 28 \[r2\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/73#-KCRiVu_aB2LDIPt4-7b-r2-28:-KCUq_vXbmaxSHArE7ai:-499146009) ([raw file](https://github.com/openprocurement/robot_tests/blob/ebc5124ba6a742d4ed6d2c58467e1bd5064fa569/op_robot_tests/tests_files/contract.robot#L28)):*
Done.

---

*[op_robot_tests/tests_files/contract.robot, line 35 \[r2\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/73#-KCRiVu_aB2LDIPt4-7b-r2-35:-KCUqaILBuKA6S2hVDAi:-499146009) ([raw file](https://github.com/openprocurement/robot_tests/blob/ebc5124ba6a742d4ed6d2c58467e1bd5064fa569/op_robot_tests/tests_files/contract.robot#L35)):*
Done.

---

*[op_robot_tests/tests_files/qualification.robot, line 13 \[r2\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/73#-KCGKJYOIMrasklbU9L1-r2-13:-KCUrWVRmKMxsP2GyqK3:-499146009) ([raw file](https://github.com/openprocurement/robot_tests/blob/ebc5124ba6a742d4ed6d2c58467e1bd5064fa569/op_robot_tests/tests_files/qualification.robot#L13)):*
Done.

---

*[op_robot_tests/tests_files/brokers/openprocurement_client.robot, line 365 \[r2\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/73#-KCGKJYOIMrasklbU9Kz-r2-365:-KCUrNcI4RwwpBU_HW82:572646580) ([raw file](https://github.com/openprocurement/robot_tests/blob/ebc5124ba6a742d4ed6d2c58467e1bd5064fa569/op_robot_tests/tests_files/brokers/openprocurement_client.robot#L365)):*
У розширеному тесті процесу кваліфікації.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/openprocurement/robot_tests/73)*

**Comment by Leits** [comment]
https://github.com/openprocurement/robot_tests/pull/86

**Pull request merged**